t/lib/HTFeed/Test/TempDirs.pm

```diff
@@ -59,6 +59,11 @@ sub setup_example {
   foreach my $dirtype ($self->repo_dirtypes) {
     $self->{$dirtype} = tempdir("$tmpdir/feed-test-$dirtype-XXXXXX");
     set_config($self->{$dirtype},'repository',$dirtype);
+    my $storage_classes = get_config('storage_classes');
```

Comment:
Hmm, this is a bit messy.. I had forgotten about `TempDirs` assuming a global config. I think this is OK for use in `collate.t` (where we have to set the global storage class config anyway) but we could perhaps consider instead making it `collate.t`'s responsibility to call `set_config(...,'storage_classes')`...

Reply:
`storage.t` also relies on the `TempDirs.pm` hack, so actually it seems reasonable to me to keep the responsibility for mucking with the global config there.

Reply:
Right, I was hoping that `storage.t` could avoid having to fuss with the global config at all (i.e. have it only rely on storage classes constructed with the particular config they need rather than having to look in the global config)